src/main/java/com/patchworkmc/mapping/remapper/ManifestRemapper.java

```diff
@@ -0,0 +1,47 @@
+package com.patchworkmc.mapping.remapper;
+
+import net.fabricmc.tinyremapper.IMappingProvider;
+import net.fabricmc.tinyremapper.TinyRemapper;
+
+import com.patchworkmc.manifest.api.Remapper;
+
+public class ManifestRemapper implements Remapper, AutoCloseable {
+	private org.objectweb.asm.commons.Remapper asmRemapper;
+	private TinyRemapper tiny;
+
+	public ManifestRemapper(IMappingProvider mappings) {
+		// TODO: this is a copy-paste of the old tinyremapper that was used in my original AT fork.
+		// Some of these flags might not be needed.
+		this.tiny = TinyRemapper.newRemapper()
+			.withMappings(mappings)
+			.rebuildSourceFilenames(true)
+			.ignoreFieldDesc(true)
```

Comment:
Probably not needed since patchwork fills in the field desc stuff, however it's good to keep this one imo